gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -0,0 +1,655 @@
+
+    def update_member(self, context, old_member, member):
+        arg_dict = {'context': context,
+                    'old_member': old_member,
+                    'member': member,
+                    'serialize': True,
+                    'binding_key': member['pool_id']
+                    }
+        ev = self._sc.new_event(id='UPDATE_MEMBER', data=arg_dict)
+        self._sc.post_event(ev)
+
+    def delete_member(self, context, member):
+        arg_dict = {'context': context,
+                    'member': member,
+                    'serialize': True,
+                    'binding_key': member['pool_id']
+                    }
+        ev = self._sc.new_event(id='DELETE_MEMBER', data=arg_dict)
+        self._sc.post_event(ev)
+
+    def create_pool_health_monitor(self, context, health_monitor, pool_id):
+
+        arg_dict = {'context': context,
+                    'health_monitor': health_monitor,
+                    'pool_id': pool_id,
+                    'serialize': True,
+                    'binding_key': pool_id
+                    }
+        ev = self._sc.new_event(id='CREATE_POOL_HEALTH_MONITOR', data=arg_dict)
+        self._sc.post_event(ev)
+
+    def update_pool_health_monitor(self, context, old_health_monitor,
+                                   health_monitor, pool_id):
+        arg_dict = {'context': context,
+                    'old_health_monitor': old_health_monitor,
+                    'health_monitor': health_monitor,
+                    'pool_id': pool_id,
+                    'serialize': True,
+                    'binding_key': pool_id
+                    }
+        ev = self._sc.new_event(id='UPDATE_POOL_HEALTH_MONITOR', data=arg_dict)
+        self._sc.post_event(ev)
+
+    def delete_pool_health_monitor(self, context, health_monitor, pool_id):
+
+        arg_dict = {'context': context,
+                    'health_monitor': health_monitor,
+                    'pool_id': pool_id,
+                    'serialize': True,
+                    'binding_key': pool_id
+                    }
+        ev = self._sc.new_event(id='DELETE_POOL_HEALTH_MONITOR', data=arg_dict)
+        self._sc.post_event(ev)
+
+    def agent_updated(self, context, payload):
+        """Handle the agent_updated notification event."""
+        arg_dict = {'context': context,
+                    'payload': payload}
+        ev = self._sc.new_event(id='AGENT_UPDATED', data=arg_dict)
+        self._sc.post_event(ev)
+
+
+"""Implements event handlers and their helper methods.
+
+Object of this class is registered with the event class of core service
+controller. Based on the event key, handle_event method of this class is
+invoked by core service controller.
+
+"""
+
+
+class LBaaSEventHandler(nfp_poll.PollEventDesc):
+    instance_mapping = {}
+
+    def __init__(self, sc, drivers, rpcmgr, nqueue):
+        self._sc = sc
+        self.drivers = drivers
+        self._rpcmgr = rpcmgr
+        self.nqueue = nqueue
```

Comment:
This "nqueue" is not needed anymore. Please remove it. Check generic config agent for example.